packages/extension-ui/src/components/Dropdown.tsx

```diff
@@ -90,7 +90,7 @@ export default styled(Dropdown)`
     position: absolute;
     top: 50%;
     transform: translateY(-50%);
-    right: 5px;
+    right: 12px;
```

Comment:
Does it improve something or was this change accidental? This Dropdown is used only in Settings.

Reply:
It was intentional, the arrow was way to close to the edge